VisualStudio.gitignore

```diff
@@ -293,3 +293,6 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+
+# Supercharger
+.vs/Supercharger/
```

Comment:
Ah sure... Sorry.. I'll check why all Supercharger files were added to my solution.
;)